pyogrio/tests/test_geopandas_io.py

```diff
@@ -187,40 +189,109 @@ def test_read_fids_force_2d(test_fgdb_vsi):
 
 
 @pytest.mark.parametrize(
-    "driver,ext",
-    [
-        ("ESRI Shapefile", "shp"),
-        ("GeoJSON", "geojson"),
-        ("GeoJSONSeq", "geojsons"),
-        ("GPKG", "gpkg"),
-    ],
+        "ext",
+        [".fgb", ".geojson", ".geojsons", ".gpkg", ".json", ".shp", ],
```

Comment:
We could remove the `.` here in every value, and add it once where the path is created?

Reply:
Possibly this is a matter of preference, not sure, but when working with extensions all (python) libraries treat the '.' as a part of the extension (Eg. os.path.splitext(), pathlib.Path.suffix).

So to me it is more consistent (and thus logical) and less error prone (eg. when doing comparisons between extensions) to consitently keep the point in the ext. 

But if you have an explicit preference I will obviously change it.

Reply:
Sorry, saw this after adding my other comment above.  I don't have a strong opinion on it and consistency with other packages is an argument for how you have it.